pkg/plugin/aggregation/handler.go

```diff
@@ -67,17 +72,27 @@ func NewHandler(resultsCallback func(*plugin.Result, http.ResponseWriter)) http.
 	return handler
 }
 
-func (h *Handler) resultsHandler(w http.ResponseWriter, r *http.Request) {
-	logRequest(r)
-	vars := mux.Vars(r)
-
+func resultFromRequest(r *http.Request, muxVars map[string]string) *plugin.Result {
```

Comment:
Intend to have a test for this logic; just to ensure we pull/default values correctly.

Reply:
Just added this test.